sql/core/src/test/resources/test-data/unparseable_decimal.csv

```diff
@@ -0,0 +1,2 @@
+money
```

Comment:
Could you write the file in the test like:
```scala
    withTempPath { path =>
      Seq(
        "money",
        "$92,807.99"
      ).toDF("data")
        .coalesce(1)
        .write.text(path.getAbsolutePath)
```
see example in CSVSuite. We don't need unnecessary dependencies in the resource folder if we can easily form the intput.